### ISSUE TITLE
When starting `webui` it crashes with SIGSEGV

### ISSUE THREAD

**Issue by tmakar** [ElektraInitiative/libelektra#4884]
## Steps to Reproduce the Problem

> ℹ️  `elektrad` is running successfully on port `33333`.

1. `cd src/tools`
2. `cd webui`
3. `npm install`
4. `npm start`

## Expected Result

`WebUI` should start.

## Actual Result

```text
  webd:info @elektra-web/webd v2.0.0 starting +0ms
  
  webd:error error while starting @elektra-web/webd: [Error [Elektra Error]: 
  Command failed: "kdb" export "user:/sw/elektra/web/#0/current/instances" 
  yajl "/tmp/elektra-web.1679136074943.buffer.json"  
  Sorry, I crashed by the signal SIGSEGV This should not have happened!  
  Please report the issue at https://issues.libelektra.org/ ] 
  { details: 'Command failed: "kdb" export "user:/sw/elektra/web/#0/current/instances" 
  yajl "/tmp/elektra-web.1679136074943.buffer.json"\n' + '\n' + 
  'Sorry, I crashed by the signal 
  SIGSEGV\n' + 'This should not have happened!\n' + '\n' + 
  'Please report the issue at https://issues.libelektra.org/\n' } 
```

## System Information

- Elektra: master
- Operating System: macOS

KDB_VERSION: 0.9.14
SO_VERSION: 5

**Comment by tmakar** [comment]
```sh
kdb export user:/sw/elektra/web/#0/current/instances yajl /tmp/elektra-web.1679136074943.buffer.json
```

Output:
```text
Sorry, I crashed by the signal SIGSEGV
This should not have happened!
```

**Comment by tmakar** [comment]
Seems that this code causes an error:
```cpp
ModulesPluginDatabase pluginDatabase;
PluginSpec provides = pluginDatabase.lookupProvides (format);
```

Probably it can not find any provider named `yajl`.

**Comment by tmakar** [comment]
When compiling / installing Elektra, I have run the following commands:
```sh
mkdir build && cd build
cmake -DPLUGINS="dump;resolver_fm_hpu_b;yajl;list;spec" -DTOOLS="kdb;web" ..
make
sudo make install
```

Is there anything I missed? Can you maybe check as well? 🤔 
@atmaxinger @kodebach

**Comment by tmakar** [comment]
ahh, I have now seen the following 
```text
Exclude plugin yajl because yajl (libyajl-dev 1 or 2) not found
```

> Examined from the output of `cmake -DPLUGINS="dump;resolver_fm_hpu_b;yajl;list;spec" -DTOOLS="kdb;web" ..`

**Comment by tmakar** [comment]
Is there any possibility to make this more visible when running `cmake`? In this case there was so much output and I have overseen this as it was just logged as info. Can we somehow log this as `warning` in case someone has explicitly set the `PLUGINS` via `-DPLUGINS` and one could not be found?

@atmaxinger @kodebach

**Comment by kodebach** [comment]
As a solution to this specific issue, I would suggest we change `kdb  export` so it prints a nicer error message along the lines of "Couldn't find plugin `yajl`". Changing CMake doesn't really help, because you might have install a pre-built version of Elektra that comes without `yajl` or the `libelektra-yajl.so` was removed after installing.

However, that is a task for @hannes99 for the new C implementation of `kdb` I'd say. I crated #4886.

Regarding CMake changes: See #4885.
